README.md

```diff
@@ -73,6 +73,28 @@ Guide](https://solr.apache.org/guide/solr/latest/getting-started/solr-tutorial.h
 - Slack: Solr Community Channel.  Sign up at https://s.apache.org/solr-slack
 - IRC: `#solr` on [libera.chat](https://web.libera.chat/?channels=#solr)
 
+## Developer Documentation
+
+ Learn more about developing Solr by reading through the developer docs in [./dev-docs](./dev-docs) source tree or building Solr from source [./dev-docs/solr-source-code.adoc](./dev-docs/solr-source-code.adoc)
+
+### Quickstart
+
+Solr uses [Gradle](https://gradle.org/) for its build system. Here are some useful hints to build and run Solr locally:
+
+- To build a Solr dev distribution:
+
+```
+./gradlew dev
+```
+
+- To run the Solr dev distribution locally:
+
+```
+cd ./solr/packaging/build/dev
+bin/solr start
```

Comment:
As a first-time user it is better to recommend starting in cloud mode. This lets the user actually create a new collection from the Admin UI instead of being stuck.
```suggestion
bin/solr start -c
```